wrappers/python/pybackend.cpp

```diff
@@ -188,9 +188,10 @@ PYBIND11_MODULE(NAME, m) {
         .value("auto_rx_sensitivity", RS2_OPTION_AUTO_RX_SENSITIVITY)
         .value("transmitter_frequency", RS2_OPTION_TRANSMITTER_FREQUENCY)
         .value("vertical_binning", RS2_OPTION_VERTICAL_BINNING)
-        .value("receiver sensitivity", RS2_OPTION_RECEIVER_SENSITIVITY)
-        .value("exposure limit toggle", RS2_OPTION_AUTO_EXPOSURE_LIMIT_TOGGLE)
-        .value("gain limit toggle", RS2_OPTION_AUTO_GAIN_LIMIT_TOGGLE)
+        .value("receiver_sensitivity", RS2_OPTION_RECEIVER_SENSITIVITY)
+        .value("exposure_limit toggle", RS2_OPTION_AUTO_EXPOSURE_LIMIT_TOGGLE)
+        .value("gain_limit_toggle", RS2_OPTION_AUTO_GAIN_LIMIT_TOGGLE)
```

Comment:
should the gain limit and exp limit remain here?

Reply:
No harm here..
If we will deprecate it we will consider removing it